magmi/inc/magmi_engine.php

```diff
@@ -407,6 +408,10 @@ public function getExceptionTrace($tk, &$traces)
                     $trstr .= "\n";
                 }
             }
+            // simple max trace depth fix
+            if (++$counter % 20) {
```

Comment:
Have you tested the code?
By reading it looks like the if will break on every counter value except 20 and multiplications of 20, right? Shouldn't there be a negation here?
1 % 20 == 1 => true
2 % 20 == 2 => true
19 % 20 == 19 => true
20 % 20 == 0 => false
21 % 20 == 1 => true

Reply:
You're right, sorry! This should be compared against zero.